scripts/find_outdated_packages_pypi.py

```diff
@@ -0,0 +1,114 @@
+#! /usr/bin/env python
```

Comment:
Could we reformulate this to use `conda-execute` and pull in dependencies that way?